driver/normalizer/strconv_test.go

```diff
@@ -0,0 +1,41 @@
+package normalizer
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUnquoteSingle_NullGo(t *testing.T) {
```

Comment:
I would rather propose a single table test here, instead of multiple identical tests

Reply:
addressed in 9799217d7f5ec25b70f4db299048a94c488da6c0